napari/_qt/qt_main_window.py

```diff
@@ -788,6 +786,11 @@ def add_function_widget(
         """
         from magicgui import magicgui
 
+        magic_kwargs = {
+            'call_button': "run",
```

Comment:
i often feel like `auto_call` and `call_button` are weird together, so we should probably pick one for now.  This is likely the hardest one for us to "guess" about.  Though it's no fun for short running functions, using `auto_call` can stink for long running functions.  So i think we might want to use `call_button` for now, and think about how we could pick between them (or sneak it into the annotation somehow)

Reply:
We can default to `call_button` for now, it is safer